Update documentation (Fall 2023)

EthanThatOneKid opened the pull request:
Resolves #864.

EthanThatOneKid committed:
Update GETTING_STARTED.md

EthanThatOneKid committed:
Update GETTING_STARTED.md

EthanThatOneKid committed:
Merge branch 'main' into fix/864

EthanThatOneKid committed:
Merge branch 'main' into fix/864

EthanThatOneKid committed:
Fix heading capitalization

EthanThatOneKid committed:
Add "Updating board member data"

Edit "Custom workflows", add "Updating board member data"

EthanThatOneKid committed:
Merge branch 'main' into fix/864

EthanThatOneKid committed:
Merge branch 'main' into fix/864

EthanThatOneKid reviewed (commented):
Should be good to go. This PR has been sitting for so long that I lost track of why this PR was not merged.

EthanThatOneKid committed:
Merge branch 'main' into fix/864

k4rni commented on GETTING_STARTED.md:
```diff
@@ -1,50 +1,50 @@
-1. Create a fork of [acmcsuf.com](https://github.com/EthanThatOneKid/acmcsuf.com), similar to <https://github.com/jaasonw/acmcsuf.com>
-1. Create an [access token](https://scribehow.com/shared/Generating_a_personal_access_token__eMf0UJYaS5OcGrO_lJTpGg)
-1. Follow [these instructions](https://github.com/EthanThatOneKid/acmcsuf.com/blob/main/CONTRIBUTING.md#setting-up-your-env-file) to add it to your dev environment
+1. Create a fork of [acmcsuf.com](https://github.com/EthanThatOneKid/acmcsuf.com), similar to <https://github.com/diamondburned/acmcsuf.com>
```
You added a period at the end of every bullet point, did you want to add one here too?

EthanThatOneKid replied:
Since these are phrased like complete sentences, I agree with you that a period is appropriate.

EthanThatOneKid replied:
Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/865/commits/24e54d0e8a43b101bdc2a973add4ec1206957805>.

k4rni commented on GETTING_STARTED.md:
```diff
@@ -1,50 +1,50 @@
+   1. Examples of bad PRs:
+      1. [https://acmcsuf.com/pull/359](https://github.com/EthanThatOneKid/acmcsuf.com/pull/359)
+      1. [https://acmcsuf.com/pull/357](https://github.com/EthanThatOneKid/acmcsuf.com/pull/357)
+1. Request a code review from someone by first contacting them and then assigning them to your PR using the "Reviewers" panel.
```
Should we mention that reviewers have to review changes within 48 hours?

EthanThatOneKid replied:
Good idea. This is a fitting location to mention that common courtesy is to review within 24 hours if possible. The author of the PR should also alert their reviewer in some way other than GitHub to make them aware that there is a PR awaiting their review.

EthanThatOneKid replied:
Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/865/commits/82d1a77b696bccfc0117a29fea76c71565501cdd>.

k4rni reviewed (approved):
A couple nits but overall it looks great!

EthanThatOneKid committed:
Merge branch 'main' into fix/864

EthanThatOneKid committed:
Merge branch 'fix/864' of https://github.com/EthanThatOneKid/acmcsuf.com into fix/864

EthanThatOneKid committed:
Include periods on complete sentences

Resolves <https://github.com/EthanThatOneKid/acmcsuf.com/pull/865#discussion_r1318067462>.

EthanThatOneKid committed:
Fix "Getting started" title

EthanThatOneKid committed:
Getting started: clarify review cycle

Resolves <https://github.com/EthanThatOneKid/acmcsuf.com/pull/865#discussion_r1318068193>.

k4rni reviewed (commented):
Nice 😎
